src/extensions/default/JavaScriptCodeHints/npm-shrinkwrap.json

```diff
@@ -0,0 +1,122 @@
+{
+  "name": "brackets-javascript-code-hints",
+  "dependencies": {
+    "acorn": {
+      "version": "3.3.0",
+      "from": "acorn@3.3.0",
+      "resolved": "https://registry.npmjs.org/acorn/-/acorn-3.3.0.tgz"
+    },
+    "buffer-shims": {
+      "version": "1.0.0",
+      "from": "buffer-shims@>=1.0.0 <2.0.0",
+      "resolved": "https://registry.npmjs.org/buffer-shims/-/buffer-shims-1.0.0.tgz"
+    },
```

Comment:
yes - that's how shrinkwrap works, it's generated by `npm shrinkwrap` command, I'd write more about it but it's easy to google